R/experiments.R

```diff
@@ -1,9 +1,36 @@
 ## The following are the default values that will be used for handling MAE experiments
+SUPPORTED_EXPERIMENTS <-
+  c(sa = "single-agent", combo = "combination")
 EXPERIMENT_GROUPS <-
```

Comment:
@bczech , @j-smola, 
maybe we can go that way?: 
```SUPPORTED_EXPERIMENTS <-
  c(sa = "single-agent", combo = "combination", cd = "co-dilution")
``` 

I'm not sure, if `cd = "co-dilution" should be added now as it's currently not supported, right?

Reply:
Maybe as comment?

Reply:
I agree with:
`  c(sa = "single-agent", combo = "combination", cd = "co-dilution")`

We can keep the co-dilution there since it is used in the gDRcore i.e. in data_type functions

Reply:
done in dc91a57d8556b61fa31b31287b04739d416b7ea1.